lib/msal-node/src/client/ManagedIdentityApplication.ts

```diff
@@ -82,6 +84,8 @@ export class ManagedIdentityApplication {
 
         this.cryptoProvider = new CryptoProvider();
 
+        this.correlationId = this.cryptoProvider.createNewGuid();
```

Comment:
Is there an API for app developer to pass in his own correlation ID?

Reply:
It appears we didn't implement that in Managed Identity. Unlike all other msal-node clients, Managed Identity creates the request (```ManagedIdentityRequest```) inside of the ```acquireToken``` call, instead of passing in the request. We currently only pass in ```ManagedIdentityRequestParams``` (forceRefresh?, resource) to the ```aquireToken``` call.

Thoughts?

Reply:
https://github.com/AzureAD/microsoft-authentication-library-for-js/issues/6919